usecases/config/config_handler.go

```diff
@@ -52,6 +52,11 @@ const (
 	DefaultAlpha = float64(0.75)
 )
 
+const (
+	HybridRankedFusion = iota
+	HybridRelativeScoreFusion
+)
```

Comment:
Why did you re-introduce those settings here? now there are in 2 places here and in [hybrid_search.go](https://github.com/weaviate/weaviate/blob/stable/v1.20/adapters/handlers/graphql/local/get/hybrid_search.go#L23-L26)

Reply:
I would use those from [hybrid_search.go](https://github.com/weaviate/weaviate/blob/stable/v1.20/adapters/handlers/graphql/local/get/hybrid_search.go#L23-L26) and remove them from this file

Reply:
I removed them from hybrid_search.go, lost that change during rebasing.

They can't stay in that file, because of cyclic imports

Reply:
Then let's think of a better place, config_hander.go holds the settings that are `config specific` not `argument config specific`

Reply:
it holds the default value for alpha, that is why I moved it there 😅 But if you can thing of a better place let me know

Reply:
and if you move those consts to `common_filters` would that also create cyclic imports?